### DISCUSSION TITLE
Are we translating only MD files?

### DISCUSSION THREAD

**Discussion by saranshkataria** [reactjs/reactjs.org-translation#241] [Team Posts]
Hey, I am assuming that we are translating only translating MD files? There is some text in the javascript files which shows up on the website as well, are we touching those or not?

Example: the "Take the tutorial" in index.js

**Comment by GasimGasimzada** [comment]
We have translated code editor texts, header and footer texts, and some template texts (e.g homepage).

**Comment by sorousht** [comment]
@saranshkataria According to [Universal Style Guide](https://github.com/reactjs/reactjs.org-translation/edit/master/style-guide.md), we are allowed to translate comments and the strings that do not change a code logic. So I think you can translate the example with no worries.

**Comment by saranshkataria** [comment]
Awesome. Thanks for the quick inputs!
